groovy/postInit/chemistry/ChemistryOverhaul.groovy

```diff
@@ -4573,6 +4367,63 @@ BR.recipeBuilder()
         .EUt(30)
         .buildAndRegister()
 
+// Ethylbenzene
+
+CSTR.recipeBuilder()
+        .fluidInputs(fluid('benzene') * 50)
+        .fluidInputs(fluid('ethylene') * 50)
+        .notConsumable(fluid('sulfuric_acid') * 50)
+        .fluidOutputs(fluid('ethylbenzene') * 50)
+        .duration(5)
+        .EUt(120)
+        .buildAndRegister()
+
+// Nitromethane
+
+CSTR.recipeBuilder()
+.fluidInputs(fluid('propane') * 50)
+.fluidInputs(fluid('nitric_acid') * 150)
+.fluidOutputs(fluid('diluted_nitromethane') * 200)
```

Comment:
doesn't this reaction produce a mixture of nitroalkanes?

Reply:
this is a simplification of the process as so to not produce useless byproducts